conda/config.py

```diff
@@ -171,7 +171,7 @@ def pkgs_dir_from_envs_dir(envs_dir):
 
 # ----- default environment prefix -----
 
-_default_env = os.getenv('CONDA_DEFAULT_ENV')
+_default_env = os.getenv('CONDA_ACTIVE_ENV')
```

Comment:
Incomplete rename; probably will be reverted.  Thanks for pointing it out, regardless.